moveit_ros/planning_interface/common_planning_interface_objects/src/common_objects.cpp

```diff
@@ -99,7 +100,7 @@ namespace planning_interface
 std::shared_ptr<tf2_ros::Buffer> getSharedTF()
 {
   SharedStorage& s = getSharedStorage();
-  boost::mutex::scoped_lock slock(s.lock_);
+  std::scoped_lock slock(s.lock_);
```

Comment:
Foxy is still on c++14. This needs to be a unique_lock or lock_guard

Reply:
Boost's scoped_lock is just an alias for unique_lock, so I'm going to pick that one for now